website/project/model.py

```diff
@@ -572,6 +573,10 @@ class Node(GuidStoredObject, AddonModelMixin):
     registered_user = fields.ForeignField('user', backref='registered')
     registered_schema = fields.ForeignField('metaschema', backref='registered')
     registered_meta = fields.DictionaryField()
+    is_retracted = fields.BooleanField(default=False)
+    retracted_justification = fields.StringField()
```

Comment:
Good question. They could easily write a paragraph justifying this. I'll raise the question with @bnosek and @saradbowman